src/Clock.tsx

```diff
@@ -0,0 +1,59 @@
+import { Component } from 'react';
+
+type Props = {
+  clockName: string
+};
+
+type State = {
+  today: string
+};
+
+export class Clock extends Component<Props> {
+  state: Readonly<State> = {
+    today: new Date().toUTCString().slice(-12, -4),
+  };
+
+  runClock = 0;
+
+  componentDidMount() {
+    this.runClock = window.setInterval(() => {
+      this.setState({
+        today: new Date().toUTCString().slice(-12, -4),
+      });
+      /* eslint-disable-next-line no-console */
+      console.info((this.state.today));
+    }, 1000);
+  }
+
+  componentDidUpdate(prevProps: Readonly<Props>) {
+    const { clockName } = this.props;
+
+    if (prevProps.clockName !== clockName) {
+      /* eslint-disable-next-line no-console */
+      console.debug(`Renamed from ${prevProps.clockName} to ${clockName}`);
+    }
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.runClock);
+  }
+
+  render() {
+    const { today } = this.state;
+    const { clockName } = this.props;
+
+    return (
+      <div className="Clock">
+        <strong className="Clock__name">
+          {clockName}
+        </strong>
+
+        {' time is '}
```

Comment:
Wrap with span and remove empty lines

Reply:
code stops working when changed
![image_2023-06-18_12-02-53](https://github.com/mate-academy/react_clock/assets/83349437/d1f51292-a701-4f8b-b078-aa857875cc83)